fml/platform/android/message_loop_android.cc

```diff
@@ -63,23 +88,13 @@ void MessageLoopAndroid::Run() {
   FML_DCHECK(looper_.get() == ALooper_forThread());
 
   running_ = true;
-
-  while (running_) {
-    int result = ::ALooper_pollOnce(-1,       // infinite timeout
-                                    nullptr,  // out fd,
-                                    nullptr,  // out events,
-                                    nullptr   // out data
-    );
-    if (result == ALOOPER_POLL_TIMEOUT || result == ALOOPER_POLL_ERROR) {
-      // This handles the case where the loop is terminated using ALooper APIs.
-      running_ = false;
-    }
-  }
+  LooperPrepare();
```

Comment:
does this means that `ALooper_forThread()` returns `null`? or are we calling prepare unnecessarily?

Reply:
nvm, this is initializing the state in the JVM, right?  could we add comments?

Reply:
Done.